src/Tickets/Emails/Admin/Emails_Tab.php

```diff
@@ -172,4 +224,126 @@ public function get_fields(): array {
 		 */
 		return apply_filters( 'tec_tickets_emails_settings_fields', $fields );
 	}
+
+	/**
+	 * Check if currently editing email.
+	 * 
+	 * @since TBD
+	 *
+	 * @param Email_Abstract $email
+	 * 
+	 * @return boolean
+	 */
+	public function is_editing_email( $email = null ) {
+		if ( ! $this->is_on_section( static::$edit_section_slug ) ) {
+			return false;
+		}
+
+		// Get `tc-edit` query string from URL.
+		$editing_email  = tribe_get_request_var( static::$key_edit_id );
+
+		// If email wasn't passed, just return whether or not string is empty.
+		if ( empty( $email ) ) {
+			return ! empty( $editing_email );
+		}
+
+		// Otherwise, return whether or not supplied email is being edited.
+		return $email->id === $editing_email;
+	}
+
+	/**
+	 * Get email settings.
+	 *
+	 * @since TBD
+	 * 
+	 * @return array|null Settings array
+	 */
+	public function get_email_settings() {
+		$email_id  = tribe_get_request_var( static::$key_edit_id );
+		$email = tribe( Email_Handler::class )->get_email_by_id( $email_id );
+		
+		$back_link = [[
+			'type' => 'html',
+			'html' => $this->get_template()->template( 'back-link', [
+				'text' => __( 'Back to Email Settings', 'event-tickets' ),
+				'url'  => $this->get_url(),
+			], false ),
+		]];
+
+		if ( ! $email ) {
+			return array_merge( $back_link, [
+				[
+					'type' => 'html',
+					'html' => '<p>' . esc_html__( 'Invalid email id selected.', 'event-tickets') . '</p>',
+				]
+			] );
+		}
+
+		$hidden_fields = [[
```

Comment:
We should start the second array in a new line. We can simplify the checks by just having the check on `section`, we don't want to validate if it's a valid email section. It would work the same way as we have it for Tickets Commerce and the payment gateways (in that case `tc-section`, that we'll probably move to `section` in a future)